### DISCUSSION TITLE
Which Home Assistant ?

### DISCUSSION THREAD

**Discussion by thesebastianf** [tteck/Proxmox#3620] [General]
Hey, I am pretty new to Proxmox and would like to learn the pros and cons or maybe general advice for "best" HA version on Proxmox ... VM, LXC,... ?

Thank you :)

**Comment by Mellowlynx** [comment]
It really depends on what you plan to do and add.
I use the OS VM, so I can add a ZigBee & Z-Wave dongle easy. (But this is also possible with the LXC.)
Also, all management (updates) is done by HA itself and through the UI, so great to start with.
The Core LXC provides a simpler, but less flexible and scalable solution.

If you start small with the Core LXC, you should be able to migrate a HA backup to the OS VM.
Note, I'm not 100% sure, so to be sure you should try it. This will also give you the ability to see both installations.
